templates/base.html

```diff
@@ -102,6 +103,7 @@
         {% endif %}
         <script src="//code.jquery.com/jquery.min.js"></script>
         <script src="//netdna.bootstrapcdn.com/twitter-bootstrap/2.3.2/js/bootstrap.min.js"></script>
+        <script src="//unpkg.com/applause-button/dist/applause-button.js"></script>
```

Comment:
Just like CSS, also add the JS file to the source. So that we do not have to make request to a third party server to fetch the JS file.

See how tipuesearch code is added to the project.

Reply:
done.